frontend/actions.js

```diff
@@ -122,20 +129,29 @@ export default ({setState, getState}) => {
     }
   }
 
-  const isSendFormFilledAndValid = (state) => {
-    return (state.sendAddress.fieldValue !== '' && state.sendAmount.fieldValue !== '' &&
-  state.sendAddress.validationError === '' && state.sendAmount.validationError === '')
-  }
-
   const validateFee = (state) => {
-    const sendAmount = parseFloat(state.sendAmount.fieldValue)
-    const transactionFee = state.transactionFee / 1000000
-    const balance = state.balance / 1000000
-    if (sendAmount + transactionFee > balance) {
-      setState({sendAmount: Object.assign({}, state.sendAmount, {validationError: {code: 'SendAmountInsufficientFundsForFee', params: {balance, transactionFee}}})})
+    const sendAmountForTransactionFee = state.sendAmountForTransactionFee
+    const transactionFee = state.transactionFee
+    const balance = state.balance
+
+    if (sendAmountForTransactionFee + transactionFee > balance) {
+      setState({
+        sendAmount: Object.assign({}, state.sendAmount, {
+          validationError: {code: 'SendAmountInsufficientFunds', params: {balance: state.balance}},
+        }),
+      })
     }
   }
 
+  const isSendFormFilledAndValid = (state) => {
```

Comment:
Is there any reason we use `return` instead of the shorthand notation
`(state) => (... && ... && ... && ...)` ?

Reply:
fixed